lib/assert.js

```diff
@@ -122,8 +122,15 @@ function wrapAssertions(callbacks) {
 			}
 		},
 
-		log(text) {
-			log(this, text);
+		log(...args) {
```

Comment:
For the next few months we're still targeting Node.js 4, which doesn't support rest parameters. You'll have to use plain old `arguments` instead.